### DISCUSSION TITLE
Respect date: field in Yaml header

### DISCUSSION THREAD

**Discussion by sebo-b** [damionx7/Zettel-Notes-Documentation#224] [Feature Requests]
When date field is present in yaml header, it should be considered for calendar view and search. In addition a setting to automatically update it on save would be great.

**Comment by damionx7** [comment]
Hey, Can you share the example YAML code.

**Comment by sebo-b** [comment]
Hi,

yes, so it is simple (sorry for the screenshot):
[image.png]

Or in GitJournal format:
[image.png]

There is also GitJournal extension to VSCode to update these timestamps on every save: https://github.com/GitJournal/GitJournal/discussions/959

And, I read yesterday somewhere that Markdown All-in-one extension can also do that, but I cannot find this setting nor the source anymore - so maybe I was wrong.

Regards,
/S

**Comment by damionx7** [comment]
So this is the date format. Currently you can use `id or uid` in zettel notes as yaml front matter to change the note creation date. Read more at https://znotes.thedoc.eu.org/note/yaml-frontmatter/?h=yaml.

![date-format](https://private-user-images.githubusercontent.com/1049172/302057300-10856082-14ee-4f26-ab77-608872cae0c4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMyOTg4MTksIm5iZiI6MTczMzI5ODUxOSwicGF0aCI6Ii8xMDQ5MTcyLzMwMjA1NzMwMC0xMDg1NjA4Mi0xNGVlLTRmMjYtYWI3Ny02MDg4NzJjYWUwYzQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMDRUMDc0ODM5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MzExZjAzNDFiYmJhZWI3NDZlZGNiNTUwZDcxYmFjNDk1ZTBmODg1M2Y3NGI5NGQ0YjFkNWNiMTRjMjkyOTZiMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.Sudm12K4Ij_fkhqJEpBcEzsOaBifWv6VcVXrwiP9B5U)
